Lib/test/test_urllib.py

```diff
@@ -1537,9 +1537,10 @@ def test_quoting(self):
                          "url2pathname() failed; %s != %s" %
                          (expect, result))
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
Why is this test erroring/failing?

Reply:
I don't know. But it was expected failure before updating. So it will fails on windows without it. See 1542 line before patch.